build/npm/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "jfrog-cli-go",
   "version": "1.49.0",
-  "description": "🐸 Command-line interface for JFrog Artifactory, Bintray, Mission Control and Xray 🐸",
+  "description": "🐸 Command-line interface for JFrog Artifactory, Mission Control and Xray 🐸",
```

Comment:
"description": "🐸 Command-line interface for JFrog Artifactory, Mission Control and Xray 🐸",
-->
"description": "🐸 Command-line interface for JFrog Artifactory, Xray, Distribution and Mission Control 🐸",